src/Command/CommandBase.php

```diff
@@ -1217,16 +1217,20 @@ protected function fillMissingRequiredApplicationUuid(InputInterface $input, Out
    *
    * @throws \Psr\Cache\InvalidArgumentException
    */
-  protected function convertApplicationAliastoUuid(InputInterface $input): void {
+  protected function convertApplicationAliasToUuid(InputInterface $input): void {
     if ($input->hasArgument('applicationUuid') && $input->getArgument('applicationUuid')) {
       $application_uuid_argument = $input->getArgument('applicationUuid');
       try {
         self::validateUuid($application_uuid_argument);
       } catch (ValidatorException $validator_exception) {
         // Since this isn't a valid UUID, let's see if it's a valid alias.
         $alias = $this->normalizeAlias($application_uuid_argument);
-        $customer_application = $this->getApplicationFromAlias($alias);
-        $input->setArgument('applicationUuid', $customer_application->uuid);
+        try {
+          $customer_application = $this->getApplicationFromAlias($alias);
+          $input->setArgument('applicationUuid', $customer_application->uuid);
+        } catch (AcquiaCliException $exception) {
+          throw new AcquiaCliException("{applicationUuid} must be a valid UUID or application alias.");
```

Comment:
Is there any other reason that the enclosed code would throw an exception? Like if auth fails or there's a network error, will people get a false positive about the UUID being invalid?

I'd be a little more comfortable if we check the caught exception message before rethrowing.

Reply:
Notice that we are only catching exceptions of type AcquiaCliException. Other exceptions will not be caught.